index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <link rel="stylesheet" type="text/css" href="style.css">
+  <script type="text/javascript" src="script.js"></script>
+  <title>Carrye Gaulin</title>
+</head>
+
+<body>
+  <ul>
+    <li><a href="#Resume" class="active">Resume</a></li>
+    <li><a href="#GitHub">GitHub</a></li>
+    <li><a href="#LinkedIn">LinkedIn</a></li>
```

Comment:
_Nit_ these three should be external links to your actual linkedin, github, and resume. Also you should set them to open in a new tab ``<a href='url' target='_blank'></a>